### DISCUSSION TITLE
vite 支持整个机器人热更

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#873] [General]
尤老板写的什么怪物

![image](https://user-images.githubusercontent.com/9006264/183231856-43c4ee63-fa24-4870-a945-f3127f692b20.jpeg)

**Comment by Arondight** [comment]
vite 确实是支持的，之前用的还挺方便的，后端改了代码 vite 会重新部署，页面也会刷新，前一阵我记得 vite3 出来了，也没细看

**Reply by mark9804** [reply]
3.0 我能感知到的变化是更快了，还有就是默认端口从 3000 → 5173 （，悄悄修了一个 `-host` 选项不能显示开发服务器运行 IP 的问题），[import.meta.glob](https://cn.vitejs.dev/guide/features.html#glob-import) 部分的更新没看

Rollup 接下来几个月要出 V4 了，vite 应该也会跟着出 vite 4
